/**:
  ros__parameters:
    frame_id: "map"
    pose_frame_id: ""
    lookahead_distance: 1.0
    cmd_vel_distance_rate: 0.2
    vel_limit:
      linear:
        x: 1.0
        y: .inf
        z: .inf
      angular:
        x: .inf
        y: .inf
        z: 1.0
    accel_limit:
      linear:
        x: 2.0
        y: .inf
        z: .inf
      angular:
        x: .inf
        y: .inf
        z: 2.0
    cmd_vel_timeout: 10.0
